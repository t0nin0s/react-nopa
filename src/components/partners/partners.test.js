import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import Partners from './partners'

test('Partners component renders as expected in homepage', () => {
  const component = shallow(<Partners />)
  const tree = toJson(component)
  expect(tree).toMatchSnapshot()
})
