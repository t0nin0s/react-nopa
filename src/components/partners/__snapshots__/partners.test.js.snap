// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Partners component renders as expected in homepage 1`] = `
<section
  className="section section-tertiary"
>
  <div
    className="wrapper"
  >
    <div
      className="container centered"
    >
      <h1>
        Our partners:
      </h1>
      <ul>
        <li
          className="partnerItem"
        >
          <img
            alt=""
            src={Object {}}
          />
        </li>
        <li
          className="partnerItem"
        >
          <img
            alt=""
            src={Object {}}
          />
        </li>
        <li
          className="partnerItem"
        >
          <img
            alt=""
            src={Object {}}
          />
        </li>
        <li
          className="partnerItem"
        >
          <img
            alt=""
            src={Object {}}
          />
        </li>
      </ul>
    </div>
  </div>
</section>
`;
